.gitignore

```diff
@@ -0,0 +1,7 @@
+
+
+package-lock.json
+package.json
+node_modules
+test.md
+.github
```

Comment:
Thank you for your patient explanation. I was nervous before I got your code review but after seeing your clear explanations I feel excited to resubmit my project.

Reply:
@skyv26

Reply:
Can I remove both package-lock.json and package.json files from my project folder? @skyv26 .

Reply:
@HtetWaiYan7191 please remove both from `gitignore` but put at the same level like other files like `index.html` etc. when you try to run command `npm i` it will look for your `package.json` and generate the `package-lock.json` file automatically and also create the **node_modules** so `package.json` is very very important. At last remove the unnecessary things from `.gitignore` but don't from your project folder.

Reply:
thank you very much for your clear explanations.